### PR TITLE
LPS-98183 Add test to check when a Data Definition is deleted, all entities related are deleted too

### PR DESCRIPTION
**Pull request by marcelabc**

### CODE REVIEW

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by marcelabc** [comment]
ci:test:sf

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by jeyvison** [comment]
ci:test:relevant

File: modules/apps/data-engine/data-engine-rest-test/src/testIntegration/java/com/liferay/data/engine/rest/resource/v1_0/test/util/DataLayoutTestUtil.java [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+import java.util.HashMap;
+
+/**
+ * @author Marcela Cunha
+ */
+public class DataLayoutTestUtil {
+
+	public static DataLayout createDataLayout(
+		long groupId, long dataDefinitionId, String name) {
```
Comment (marcelabc) [inline modules/apps/data-engine/data-engine-rest-test/src/testIntegration/java/com/liferay/data/engine/rest/resource/v1_0/test/util/DataLayoutTestUtil.java]: sort

File: modules/apps/data-engine/data-engine-rest-test/src/testIntegration/java/com/liferay/data/engine/rest/resource/v1_0/test/DataDefinitionResourceTest.java [hunk 2]
```diff
@@ -47,6 +47,62 @@
 public class DataDefinitionResourceTest
 	extends BaseDataDefinitionResourceTestCase {
 
+	@Override
+	@Test
+	public void testDeleteDataDefinition() throws Exception {
+		super.testDeleteDataDefinition();
+
```
Comment (marcelabc) [inline modules/apps/data-engine/data-engine-rest-test/src/testIntegration/java/com/liferay/data/engine/rest/resource/v1_0/test/DataDefinitionResourceTest.java]: variavel do fieldname com randomString

**Pull request closed without merging**